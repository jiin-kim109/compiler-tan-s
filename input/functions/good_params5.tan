subr void holler(char c) {
	print c \n;
	
	var n := 0;
	
	while (n < c - <int>('0')) {
		print "Yoo hoo!!!!" \n;
		n := n + 1;
	}
	return;
}

main {
	call holler( '4' );
	
	var m := 0.34;
	call holler( '2' );
	
	print "all done." \n;
}
