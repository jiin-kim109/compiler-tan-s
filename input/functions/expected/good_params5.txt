4
Yoo hoo!!!!
Yoo hoo!!!!
Yoo hoo!!!!
Yoo hoo!!!!
2
Yoo hoo!!!!
Yoo hoo!!!!
all done.
